### PR TITLE
Updates for Xcode 7 beta 4

### PR DESCRIPTION
**Pull request by ikesyo**
Please see each commit for the details. :wink:

### CODE REVIEW

File: ReactiveCocoa/Swift/TupleExtensions.swift [hunk 1]
```diff
@@ -16,7 +16,7 @@ internal func repack<A>(t: (), value: A) -> (A) {
 }
 
 internal func repack<A, B>(t: (A), value: B) -> (A, B) {
```
Comment (neilpa) [inline ReactiveCocoa/Swift/TupleExtensions.swift]: Do we even need to declare the input as a tuple anymore?

**Comment by neilpa** [comment]
One quick question

File: ReactiveCocoa/Swift/TupleExtensions.swift (hunk 1 above)
Comment (ikesyo) [inline ReactiveCocoa/Swift/TupleExtensions.swift]: There was no need to use tuple type here. :boom:

Comment (ikesyo) [inline ReactiveCocoa/Swift/TupleExtensions.swift]: Besides, because `repack<A>` and `repack<A, B>` are not used, should we remove them?

Comment (neilpa) [inline ReactiveCocoa/Swift/TupleExtensions.swift]: Sure

Comment (ikesyo) [inline ReactiveCocoa/Swift/TupleExtensions.swift]: Thanks, done in 1187e78.

**Comment by neilpa** [comment]
:boom:

**Pull request merged**